src/utils/cli-utils.ts

```diff
@@ -60,3 +60,13 @@ export async function spinner(func, userMsg: string = '') {
 export function toJsonPretty(obj) {
   return JSON.stringify(obj, undefined, 2);
 }
+
+export function dateToString(date){
+  const options = {
+    year: 'numeric', month: 'short', day: 'numeric',
+    hour: 'numeric', minute: 'numeric',
+    timeZone: 'UTC',
+    timeZoneName: 'short'
+  };
+  return new Intl.DateTimeFormat('en-US', options).format(date)
```

Comment:
just an idea, as I didn't trace where it is used, but I think we should use user locale to determine the date time format not hardcode it to en-US (unless it is for sending data to the backend).